### DISCUSSION TITLE
Rethinking the u-root repository purpose

### DISCUSSION THREAD

**Discussion by MDr164** [u-root/u-root#2538] [Ideas]
The current state as of writing is that we got both the initramfs builder and the coreutils inside the repository. The busybox builder is already factored out into the gobusybox repo, so it might make sense to also move the initramfs builder out or the other way around keep the builder here and move the commands out. This way we can version those separatly. This discussion should serve as a ground to open proposals or different ideas on how to move forward.

**Comment by hugelgupf** [comment]
Happening in https://github.com/u-root/mkuimage now :)
